### PR TITLE
672 employer faqs

### PR DESCRIPTION
**Pull request by isoworg**
Adds employer FAQs page and more employer testimonials.

### CODE REVIEW

File: source/partials/_employer_testimonials.html.haml [hunk 1]
```diff
@@ -3,6 +3,12 @@
     .slide
       .container
         .profile-row
+          .copy-center
+            %blockquote ThoughtWorks usually hires just 1% of people we interview <br>but we've hired 68% of the people we interviewed from Makers Academy.
```
Comment (nikeshashar) [inline source/partials/_employer_testimonials.html.haml]: Can we say "nearly 70% of the people we interviewed from Makers Academy?"

It sounds better IMO

Comment (isoworg) [inline source/partials/_employer_testimonials.html.haml]: Nope. Studies show that a non-round number is more believable.

File: data/employer_faqs.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+- category_name: ''
+  faqs:
+    - question: What can someone do after just 12 weeks?
+      answer: "Our developers work primarily with Ruby and Javascript during the course, but the aim of our [curriculum](/curriculum/) is not to create Ruby experts - rather independent, language-agnostic problem-solvers. We believe that you’re not a good developer unless you can pick up new languages at the drop of a hat, you write clean code which is object-oriented and test-driven, and the confidence and perseverance to solve complex bugs is part of your DNA. After 12 weeks, developers are experienced in refactoring and writing understandable, object-oriented code, testing, and pair-programming. They can tackle any problem and build an incredibly good MVP."
```
Comment (nikeshashar) [inline data/employer_faqs.yml]: JavaScript

Comment (nikeshashar) [inline data/employer_faqs.yml]: > After 12 weeks, developers are experienced 

After 12 weeks, our developers are experienced

File: data/employer_faqs.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    - question: How do I know they'll be any good?
+      answer: "First, we have a [rigorous selection process](#whats-your-selection-process-like): fewer than 10% of applicants make it onto the course. Second, we have more than 600 alumni successfully working in more than 300 companies. You can read testimonials of some of our hiring partners have said about us, such as [company1](/employers/company1), [company2](/employers/company2), and [company3](/employers/company3)."
```
Comment (nikeshashar) [inline data/employer_faqs.yml]: > such as [company1](/employers/company1), [company2](/employers/company2), and [company3](/employers/company3)."

@isoworg to update

File: data/employer_faqs.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+    - question: You don't teach the technologies we use, does that matter? 
+      answer: "It doesn’t matter at all. Our developers start with Ruby because it’s a straightforward language for a beginner to grasp, and it’s quick and easy to test, but by the time they complete the course they are fully-fledged, language-agnostic problem solvers. In the final two weeks, when teams of students create their own projects independently, many choose to teach themselves entirely new languages. Over 70% of the roles our developers are hired into are non-Ruby - Java at ThoughtWorks, C# at comparethemarket.com, Node.js at Atos Consulting, Swift at Deloitte Digital, and many, many more."
```
Comment (nikeshashar) [inline data/employer_faqs.yml]: > Over 70% of the roles our developers

See discussion about TW stat

File: data/employer_faqs.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      
+    - question: Will they be able to 'hit the ground running'?
+      answer: "Our developers are super-smart, super-fast-learners - they wouldn’t have made it through the course otherwise. In that respect they’ll be able to pick up new technologies really quickly, and problem-solve independently. One of our hiring partners can get a Makers Academy developer billable in just two weeks! At the same time, they've only been coding for a few months, so will usually need some support from a more senior developer initially, to get to know the domain. We'll work with you to ensure you get the best out of your new hires as quickly as possible."
```
Comment (nikeshashar) [inline data/employer_faqs.yml]: Senior Developer

File: data/employer_faqs.yml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+    - question: Do you have any female developers?
+      answer: "We are heavily invested in overcoming gender bias in the industry - our cohorts are typically 35% women, which is practically unheard of in the industry - and we are working to increase this further. Our coaching team is a 50/50 split of male and female coaches."
+
+    - question: How do you find the best people?
```
Comment (nikeshashar) [inline data/employer_faqs.yml]: There's a variable available to insert for the number of grads stat. `stats.yml` file. Makes it consistent with the rest of the site

File: data/employer_faqs.yml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+    - question: We're outside London, can you help?
+      answer: "Absolutely! People travel from all over the country to our on-site course in London, and we also run a remote course for aspiring developers all over the world. Outside of London, we’ve been hiring for the Co-op in Manchester, Deloitte Digital and ThoughtWorks in Germany, and for [FNZ in the Czech Republic](https://blog.makersacademy.com/how-to-hire-from-coding-bootcamps/), to name a few."
```
Comment (nikeshashar) [inline data/employer_faqs.yml]: Update the link to our Medium one to bypass the 301 : https://blog.makersacademy.com/how-to-hire-from-coding-bootcamps-1868245733dc#.5zc6ci3ds

File: data/employer_faqs.yml (hunk 5 above)
Comment (isoworg) [inline data/employer_faqs.yml]: Why?

File: data/employer_faqs.yml [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+    - question: If we wanted to hire someone from you, how would that work?
+      answer: "You can either give us a call on 020 3817 8872, or complete [this short form](/employers/contact/). We’ll have a chat about whether our developers are the right fit for your opportunity, and vice versa - ideally we’d like you to visit our academy near Liverpool Street to get a proper understanding of our culture and techniques. We have a simple job description questionnaire for you to complete, and will work with you to help build your brand with our developers. This might just be through an internal advert, or we can organise events, talks or trips to the pub, so you can get to know them, and they you. If you need, we can create a tech test in a language of your choice. We do all of this on a bespoke to your hiring needs, communicating with you throughout to ensure that the shortlist we provide is right. This service is free and unlimited: you only pay if you successfully hire - it's a flat fee of £5,000."
+
```
Comment (EmiKnu) [inline data/employer_faqs.yml]: typo - 'we do all of this on a bespoke to your hiring needs'

File: data/employer_faqs.yml [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+    - question: How do you teach them?
+      answer: "'Teach' is a naughty word at Makers Academy. Instead of teachers, we have coaches, who allow the students to become self-led, independent learners. They have a hands-off approach and only intervene as a last resort: students are encouraged to pick up most concepts themselves. The course is intense - 10-12 hour days, 5-6 days per week - which means our developers are no stranger to hard work, and the focus is on encouraging best practices and good behaviours, which allow students to solve problems on their own. One of our alumni, Cambridge grad Viola, summed it up nicely in her blog: *[Unlearning how to learn](https://illogicode.wordpress.com/2016/03/14/unlearning-how-to-learn/)*."
+
```
Comment (EmiKnu) [inline data/employer_faqs.yml]: 'We don't 'teach' at Makers Academy'

**Comment by isoworg** [comment]
@EmiKnu @nikeshashar - Have made all your changes.

File: data/employer_faqs.yml [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+    - question: How do you find the best people?
+      answer: "We have a whole team dedicated to finding the best people to join Makers Academy. We market through various online channels, and have a very supportive alumni network of 600+ developers who refer potential students to us. We get a huge number of applications and accept less than 10% of these onto the course."
+
+    - question: What's your selection process like?
+      answer: "Our course is highly competitive with a rigorous selection process: less than 10% of those who apply are offered a place. Prospective students are required to learn the basics of Ruby themselves, solving problems we give them on their own before being shortlisted for interview. We select for passion, logical reasoning skills, communication, and the right attitude to learning, as well as proficiency in basic Ruby. Those successful at interview are invited to complete the four week PreCourse, which is part-time and remote. Only candidates who show the right behaviours and whose code is up to standard are offered a place to join the full 12 week course."
```
Comment (sjmog) [inline data/employer_faqs.yml]: fewer?

**Comment by sjmog** [comment]
One minor comment @isoworg then :shipit:

File: data/employer_faqs.yml (hunk 10 above)
Comment (isoworg) [inline data/employer_faqs.yml]: Can't see where you mean 'fewer' @sjmog

Comment (sjmog) [inline data/employer_faqs.yml]: ‘fewer than 10% of those’ b/c ‘those’ is discrete

Comment (isoworg) [inline data/employer_faqs.yml]: Wait, yes I can. in this case, I felt percentage was a continuum, so less sounded better. "Fewer than 10% of them" sounds purposefully grammatical and thus pretentious.

**Comment by isoworg** [comment]
@sjmog Updated to:

"fewer than 10% of applicants"

**Comment by isoworg** [comment]
Found another one...

**Pull request merged**